modules/repository-s3/src/internalClusterTest/java/org/elasticsearch/repositories/s3/S3BlobStoreRepositoryTests.java

```diff
@@ -323,6 +393,10 @@ public void maybeTrack(final String request, Headers requestHeaders) {
                 trackRequest("PutMultipartObject");
             } else if (Regex.simpleMatch("PUT /*/*", request)) {
                 trackRequest("PutObject");
+            } else if (Regex.simpleMatch("POST /*/?delete", request)) {
```

Comment:
Should there be a new test that asserts DELETION counts? At least that they are > 0.

Reply:
I think we should cover those with https://github.com/elastic/elasticsearch/pull/98083/files#diff-6c9a1598e0f9a254bd268f047a33054ba5795bb6cb5d482e9cc2d76473cff197R182 and https://github.com/elastic/elasticsearch/pull/98083/files#diff-6c9a1598e0f9a254bd268f047a33054ba5795bb6cb5d482e9cc2d76473cff197R202. But yes, let's add an explicit assertion 👍

Reply:
Thanks. I added an explicit assertion [here](https://github.com/elastic/elasticsearch/pull/98083/commits/4bae7c1ea4f4564c92e619ae1573a499745803af#diff-6c9a1598e0f9a254bd268f047a33054ba5795bb6cb5d482e9cc2d76473cff197R198)